index.html

```diff
@@ -1338,11 +1338,14 @@ <h4>Listing</h4>
                                         with the URL of the next subset.
                                     </span>
                                     <span class="rfc2119-assertion" id="tdd-reg-list-pagination-header-nextlink-attr">
-                                        The `next` link MUST have the same `limit` argument given on the initial
-                                        request as well as a zero-based `offset` argument anchored at
-                                        the beginning of the next subset.
+                                        The `next` link MUST include all arguments needed to produce the same set of
+					data and its ordering. Particularly, it MUST include the same `limit` argument
+					given on the initial request as well as a zero-based `offset` argument anchored
+					at the beginning of the next subset.
```

Comment:
I believe two assertions can't be combined. Should either separate into two or:
```suggestion
                                        The `next` link MUST include all arguments needed to produce the same set of
					data and its ordering, in particular the same `limit` argument
					given on the initial request as well as a zero-based `offset` argument anchored
					at the beginning of the next subset.
```

Reply:
I agree that in general combining assertions is not a good idea.  At the very least, it complicates testing.   At worst, it is ambiguous.  Also, include only one RFC2119 keyword per span (otherwise it complicates tooling) and give each assertion a unique id.